server/submission/definitions.js

```diff
@@ -31,7 +31,9 @@ const typeDefs = `
       noConflictOfInterest: Boolean
       files: [File]
       submissionMeta: SubmissionMeta
+      manuscriptPersons: [ManuscriptPerson!]!
```

Comment:
This needs to be tested. As far as I know the frontend logic will update the frontend manuscript (the one used in Formik) with the values it gets from the backend and later this object will be sent back to backend containing `manuscriptPersons`. I need to check that this is indeed the case and that we don't get an error for having extra stuff in the input type.

Reply:
This field is not queried for on the front-end therefore it doesn't appear in the form data.

Reply:
Right, so when the `updateSubmission` mutation runs from the frontend, the fields we specify that we want back won't include `manuscriptPersons`. Makes sense.